lua/entities/sammyservers_textscreen/cl_init.lua

```diff
@@ -68,39 +69,44 @@ local function IsInFront(entPos, plyShootPos, direction)
 	return product < 0
 end
 
+-- cache colours for rainbow text
+local colours = {}
+local colourStep = 1 -- must be multiple of 360, lowering this seems to make no performance difference
```

Comment:
Allowing users to change this could be a cool feature. As it is, there are extra modulo calls because of this, which aren't required when using a step of 1.